### DISCUSSION TITLE
duplicated `solidus_auth_devise` in Gemfile

### DISCUSSION THREAD

**Discussion by gus-dovetail** [solidusio/solidus#4539] [Troubleshooting]
In my team, we have started a new ruby/spree project and then added the solidus gem, on top that I dockerised the whole application with all the moving parts we need. 

What i've noticed is that whenever someone runs the command:
```
rails generate solidus:install \
            --migrate=true \
            --sample=true \
            --seed=true \
            --with-authentication=true \
            --payment-method=none \
            --auto-accept=true \
            --admin-email=admin@example.com \
            --admin-password=test123
```

Solidus adds the gem 'solidus_auth_devise' to the Gemfile without checking if it's already been added.

**Solidus Version**:
* solidus_core (3.1.5)
* solidus_auth_devise (2.5.4)
* solidus_api (3.1.5)
* solidus_backend (3.1.5)
* solidus_sample (3.1.5)
* solidus_social (1.4.0)
* solidus_support (0.8.1)

**Context**:
* Docker-compose.yml
* Dockerfile
* Entrypoint (here is where I run `solidus:install`)

**Comment by waiting-for-dev** [comment]
That seems a bug, yeah. Would you like to attempt a PR to fix that @gus-dovetail? :slightly_smiling_face:
